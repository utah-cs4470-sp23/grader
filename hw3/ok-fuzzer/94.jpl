write image false to ""
read image "i~" to x2V.
let Y = 78
read image "" to a
let s0D. = 35.51
read image "" to I1w_6Z.8X.
print ""
read image "#" to u5c_
print ""
