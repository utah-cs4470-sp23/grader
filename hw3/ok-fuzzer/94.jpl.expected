(WriteCmd FalseExpr "")
(ReadCmd "i~" (VarArgument x2V.))
(LetCmd (ArgLValue (VarArgument Y)) (IntExpr 78))
(ReadCmd "" (VarArgument a))
(LetCmd (ArgLValue (VarArgument s0D.)) (FloatExpr 35))
(ReadCmd "" (VarArgument I1w_6Z.8X.))
(PrintCmd "")
(ReadCmd "#" (VarArgument u5c_))
(PrintCmd "")
